packages/common-integration-tests/src/create-delete-list-cache.ts

```diff
@@ -62,9 +62,9 @@ export function runCreateDeleteListCacheTests(cacheClient: ICacheClient) {
           expect(knownCaches.length === 1).toBeTrue();
           const cache = knownCaches[0];
 
-          const expectedThroughputLimit = 1024;
-          const expectedItemSizeLimit = 1024;
-          const expectedThrottlingLimit = 100;
+          const expectedThroughputLimit = 10240;
+          const expectedItemSizeLimit = 4883;
+          const expectedThrottlingLimit = 500;
```

Comment:
I think you probably don't want to commit these?

Reply:
I do!! We have test failures otherwise as we recently bumped up these limits.